generators/csharp/codegen/src/asIs/RawClient.Template.cs

```diff
@@ -15,141 +15,141 @@ internal class RawClient(ClientOptions clientOptions)
     /// <summary>
     /// The gRPC client used to make requests.
     /// </summary>
-    public readonly RawGrpcClient Grpc = new RawGrpcClient(clientOptions);
+    public readonly RawGrpcClient Grpc = new(clientOptions);
```

Comment:
sorry for the horrible diff, prettier insisted on formatting it and i gave up on fixing it after banging my head for a while

Reply:
suggest hiding whitespace